lib/Rex/Service/FreeBSD.pm

```diff
@@ -45,11 +45,13 @@ sub ensure {
   if ( $what =~ /^stop/ ) {
     $self->stop( $service, $options );
     delete_lines_matching "/etc/rc.conf",
-      matching => qr/${service}_enable="YES"/;
+      matching => qr/${service}_enable="?[Yy][Ee][Ss]"?/;
```

Comment:
Does the `qr/${service}_enable="?[Yy][Ee][Ss]"?/` pattern has any benefits over normal case-insensitive matching?
Like:
- `qr/${service}_enable="?YES"?/i` (matches the whole expression case-insensitively)
- `qr/${service}_enable="?((?i)YES)"?/` (matches only the `YES` part case-insensitively)

If there's no specific reason to use the character list version, I would say the case-insensitive form is a bit clearer on the intent here.

Same question applies to line 54 below.